wip-asdpc-increase-collateral-utxo-age-requirement.md

```diff
@@ -0,0 +1,190 @@
+<pre>
+  WIP: WIP-ASDPC-INCREASE-COLLATERAL-UTXO-AGE-REQUIREMENT
+  Layer: Consensus (hard fork)
+  Title: Increase the age requirement for using transaction outputs as collateral 
+  Authors: Adán SDPC <adan@witnet.foundation>
+  Discussions-To: `#dev-lounge` channel on Witnet Community's Discord server
+  Status: Draft
+  Type: Standards Track
+  Created: 2022-10-26
+  License: BSD-2-Clause
+</pre>
+
+
+## Abstract
+
+This proposes an increase to the age requirement for using transaction outputs as collateral. The new required age
+will be 13440 protocol epochs, i.e. 1 week. This improvement aims at strengthening the protocol's security and
+economics.
+
+## Motivation and rationale
+
+### Current status
+
+_Collateralization_ in Witnet means that nodes that become eligible to take part in resolving a oracle request need to
+lock a certain amount of Wit coins upon publishing their commitment transaction. If their committed values pass the
+filters in the eventual _tally_ of the request, they get the collateralized coins back. If they do not pass the filters
+(aka _liars_), they lose the collateralized coins.
+
+To use a transaction output (_UTXO_) as collateral, it must have existed for more than 2,000 epochs (25 hours). We often
```

Comment:
This is currently set at 1000 epochs?

Reply:
Correct, it is set to 1000 here:

https://github.com/witnet/witnet-rust/blob/3b9e50b5248c6be996f85969581e34d9b7ff60ef/config/src/defaults.rs#L335